runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/MonitoringUtil.java

```diff
@@ -214,8 +214,8 @@ public static String getGcloudCancelCommand(DataflowPipelineOptions options, Str
     }
 
     // Assemble cancel command from optional prefix and project/job parameters.
-    return String.format("%s%s jobs --project=%s cancel %s",
-        dataflowApiOverridePrefix, GCLOUD_DATAFLOW_PREFIX, options.getProject(), jobId);
+    return String.format("%s%s jobs --project=%s cancel %s --region=%s",
```

Comment:
I'm generically confused about this ordering; is the region part of the cancel command, or can it be in the same location as the `project` flag (which is preferable to me)

Reply:
I'll need to check on that. I think it can be adjacent to the project, when the command has the --region flag (not all commands do, just most of them)